README.md

```diff
@@ -83,6 +83,15 @@ try {
 }
 ```
 
+### Running Commands
+
+Please make sure that you're on the tao root folder before run the commands:
+
+```bash
+ $ cd /path/to/your/tao/instance
+```
+
+
```

Comment:
Not sure this new section is needed... maybe it would be more meaningful showing a real command being executed. What do you think?

Reply:
Like your suggestion, I'll need to copy and paste the same instruction on the "Running worker and Initializing the queues" and "the task log container" sections or copy and paste into 10 commands, that's why I made it separately.

Reply:
What I did is to remove the Section to a simple Note. Could you check again please?